src/java/org/apache/cassandra/concurrent/LocalAwareExecutorService.java

```diff
@@ -62,15 +62,41 @@
      */
     int getPendingTaskCount();
 
+
+    /**
+     * Returns the number of core threads
+     *
+     * @return the number of core threads
+     */
+    int getCorePoolSize();
+
+    /**
+     * Sets the number of core threads.
+     */
+    void setCorePoolSize(int newCorePoolSize);
+
     /**
      * Returns the maximum allowed number of threads.
      *
      * @return the maximum allowed number of threads
      */
     int getMaximumPoolSize();
 
+    /**
+     * Sets the maximum allowed number of threads.
+     */
+    void setMaximumPoolSize(int newMaximumPoolSize);
+
     default int getMaxTasksQueued()
     {
         return -1;
     }
+
+    interface MaxWorkersListener
```

Comment:
Should the names be more consistent? `MaxWorkers` vs `MaximumPoolSize`?

Reply:
Good call, renamed.